backend/src/v2/metadata/client.go

```diff
@@ -753,6 +756,17 @@ func (c *Client) GetArtifacts(ctx context.Context, ids []int64) ([]*pb.Artifact,
 	return res.Artifacts, nil
 }
 
+// GetArtifacts ...
```

Comment:
Same as above.

Reply:
see [this comment](https://github.com/opendatahub-io/data-science-pipelines/pull/62#discussion_r1674668650)